requirements.txt

```diff
@@ -1,2 +1,3 @@
 numpy
 scipy
```

Comment:
Should we specify a minimum version of scipy?   I accidentally ran the tests in the wrong conda environment, which was python 3.7 and scipy 1.3.1, and got lots of "AttributeError: module 'scipy.linalg.lapack' has no attribute 'dpstrf'"

It was fine in my normal python3.8 environment, so I guess the default scipy there is new enough, but maybe some people still use 3.7...

Reply:
Yes, probably a good idea to specify a minimum version of each. The tutorial requires v1.4 as it depends on a new ODE solver interface, and I think that implements the LAPACK interface, so that's probably the right version.